core/src/builder/transaction/creator.rs

```diff
@@ -144,8 +146,12 @@ impl ReimburseDbCache {
                 self.operator_data = Some(
                     self.db
                         .get_operator(None, self.operator_idx as i32)
-                        .await?
-                        .ok_or(BridgeError::OperatorNotFound(self.operator_idx))?,
+                        .await
+                        .map_err(|e| eyre!("Failed to get operator data from database: {}", e))?
```

Comment:
wrong usage, please do a wrap_err with the static string instead

```suggestion
                        .wrap_err("Failed to get operator data from database")?
```